content-node/docker-compose/docker-compose.yml

```diff
@@ -41,23 +39,6 @@ services:
       - ${DATABASE_DIR}:/data/database
       - ${HOSTING_CACHE_DIR}:/data/hosting_cache
     restart: unless-stopped
-    labels:
-      - "com.centurylinklabs.watchtower.enable=true"
-    networks:
-      - content_network
```

Comment:
We might want to keep this?

Reply:
@cheran-senthil the diff is weird here, the network is still there on lines 61-62